### ISSUE TITLE
FBRef - Player ID - Duplicate IDs

### ISSUE THREAD

**Issue by kidshiva** [JaseZiv/worldfootballR#331]
Hello,

Running the fb_league_stats and I am getting duplicate player IDs for players with the same name - please see screenshot below

![image](https://github.com/JaseZiv/worldfootballR/assets/46587882/76716743-c542-4099-a40a-ad0ba572f798)

championship <- fb_league_stats(
  country = "ENG",
  gender = "M",
  season_end_year = 2024,
  tier = "2nd",
  non_dom_league_url = NA,
  stat_type = "playing_time",
  team_or_player = "player",
  time_pause = 4
)

> library(worldfootballR)
> packageVersion("worldfootballR")
[1] ‘0.6.4.7’

**Comment by tonyelhabr** [comment]
Can confirm that this is a bug. It's an edge case. I'll have to think about the best way to handle it.

**Comment by tonyelhabr** [comment]
Resolved by #332.

```r
library(dplyr)
library(worldfootballR)
packageVersion('worldfootballR')
#> [1] '0.6.4.8'
championship <- fb_league_stats(
  country = "ENG",
  gender = "M",
  season_end_year = 2024,
  tier = "2nd",
  non_dom_league_url = NA,
  stat_type = "playing_time",
  team_or_player = "player",
  time_pause = 4
)
#> Please be aware that `fb_league_stats(..., team_or_player = "player")` depends on promises, which may not always work.
#> This message is displayed once per session.

championship |> 
  filter(Player == 'Danny Ward') |> 
  select(Rk, Player, Player_Href)
#> # A tibble: 2 × 3
#>      Rk Player     Player_Href                    
#>   <int> <chr>      <chr>                          
#> 1   623 Danny Ward /en/players/d3ce0e89/Danny-Ward
#> 2   624 Danny Ward /en/players/cfb29823/Danny-Ward
```
